base/some.jl

```diff
@@ -138,10 +138,31 @@ true
     This macro is available as of Julia 1.7.
 """
 macro something(args...)
-    expr = :(nothing)
+    noth = GlobalRef(Base, :nothing)
+    something = GlobalRef(Base, :something)
```

Comment:
I wouldn't have expected to need the changes to GlobalRef here?

Reply:
I just aligned it with the way `something` is then interpolated. I don't really know why `something` is retrieved in that way instead of through interpolation, so I just aligned the usage. My shaky internal reasoning is that we don't really want someone using that macro rebinding `nothing` and then getting an entirely different object, when this is guaranteed to use `Base.nothing`.